app/models/vm_scan.rb

```diff
@@ -427,13 +405,8 @@ def delete_snapshot_by_description(mor, vm)
         server = miqVimHost[:hostname] || miqVimHost[:ipaddress]
         begin
           password_decrypt = ManageIQ::Password.decrypt(miqVimHost[:password])
-          if MiqServer.use_broker_for_embedded_proxy?(ems_type)
```

Comment:
Do we use this method anymore?  I think you deleted the callers of it in core but don't know about other repos.

Reply:
I see 2 callers removed here, but here are the rest:

```
/Users/jfrey/dev/manageiq-providers-vmware/app/models/manageiq/providers/vmware/infra_manager/vm_or_template_shared/scanning.rb:21:    args1['ems'][:use_vim_broker]     = MiqServer.use_broker_for_embedded_proxy?(args1['ems']['connect_to'])
/Users/jfrey/dev/manageiq-providers-vmware/spec/models/manageiq/providers/vmware/infra_manager/vm_scan_spec.rb:213:          allow(MiqServer).to receive(:use_broker_for_embedded_proxy?).and_return(true)
/Users/jfrey/dev/manageiq/app/models/miq_server/server_smart_proxy.rb:11:    def use_broker_for_embedded_proxy?(type = nil)
```

Reply:
:+1: the vmware PR removes the first caller, the second is an `allow to receive` in specs so I'll remove that one now, then yeah the last one is the definition so i'll kill that also